core/src/test/java/io/questdb/std/NumbersTest.java

```diff
@@ -595,29 +595,57 @@ public void testParseExplicitFloat2() throws Exception {
 
     @Test
     public void testParseFloat() throws Exception {
```

Comment:
Are these comments to be removed ?

Reply:
which comment?

Reply:
oh, testing leftover, good spot!

Reply:
fixed in https://github.com/questdb/questdb/pull/2423/commits/a2db17b3ecc9f8a98e1072be1a6bf414436ac3a6